src/hotspot/cpu/aarch64/assembler_aarch64.cpp

```diff
@@ -187,6 +187,22 @@ void Address::lea(MacroAssembler *as, Register r) const {
     zrf(Rd, 0);
   }
 
+// This encoding is similar (but not quite identical) to the encoding used
+// by literal ld/st. see JDK-8324123.
+// FIXME: PRFM should not be used with writeback modes, but the assembler
```

Comment:
FIXME: is it ok to leave this in the code?
I think we prefer filed RFE's to comments in the code that nobody will ever look at again.
You can put the RFE number in the code though.

Reply:
Thanks for your advice.

Reply:
Thanks.

Reply:
Thanks. The FIXME was there because PRFM did not support pre/post indexing and we had no check for that. 
Some guarantee checks were added and FIXME was removed.
Please check the last 3 patches.

Reply:
Thanks @sandlerwang !